zap/src/main/java/org/zaproxy/zap/extension/ascan/filters/FilterResult.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.ascan.filters;
+
+/**
+ * The result of a {@code ScanFilter#isFiltered(org.zaproxy.zap.model.StructuralNode)} method check.
+ *
+ * <p>Used to indicate if a resource was filtered and why.
+ *
+ * @author KSASAN preetkaran20@gmail.com
+ * @since 2.8.0
```

Comment:
Needs to be changed.

Reply:
You can use the placeholder `TODO add version` it'll be global swapped before release